<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDosHeader</class>
 <widget class="QDialog" name="QDosHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[ IMAGE_DOS_HEADER (DOS文件头) ] </string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_ss：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_lfarlcEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_lfanew：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [“MZ”头]*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_lfanewEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_cblp：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Bytes on last page]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Initial SP value]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_crlc：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_magicEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Relocations]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（04h）  [PE NT头地址]*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_oemidEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_oemid：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_oeminfoEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Checksum]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Overlay number]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_ssEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_res4Edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Maximum memory]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_ip：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Size of header]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_maxalloc：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [OEM identifier]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Table offset]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_cp：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cparhdrEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Initial SS value]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_crlcEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_oeminfo：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_csEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（08h）  [Reserved words]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_minalloc：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_lfarlc：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_cs：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Pages in file]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_csumEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_res[4]：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_magic：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cblpEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Initial IP value]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_minallocEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_cparhdr：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_res2Edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_spEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（14h）  [Reserved words]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_ovno：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_maxallocEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_sp：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Minimum memory]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [OEM information]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cpEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_csum：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
;
font: 12pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>e_res2[10]：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_ipEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_ovnoEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>（02h）  [Initial CS value]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
